ang/test/civicase/ActivitiesCalendar.spec.js

```diff
@@ -592,6 +624,65 @@
       });
     });
 
+    describe('"see more" link url', function () {
+      var currentRouteParams, queryParams, url;
+
+      beforeEach(function () {
+        currentRouteParams = {
+          dtab: 0,
+          foo: 'foo',
+          af: { bar: 'bar' }
+        };
+
+        initController(null, {
+          '$route': { current: { params: currentRouteParams } }
+        });
+
+        url = $scope.seeMoreLinkUrl(dates.yesterday);
+        queryParams = extractQueryStringParams();
```

Comment:
I'm not 100% sure, but I think you can use `$location` to determine the query params:

```
url = $scope.seeMoreLinkUrl(dates.yesterday);
$location.url(url);
queryParams = $location.search();
```

I know search parses the URL's query parameters (more info [here](https://docs.angularjs.org/api/ng/service/$location#search)), what I'm not sure is if `$location.url(url);` will change the URL and break the test.